src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_config_editor.cljs

```diff
@@ -195,7 +195,41 @@
                            (m "attributes")])
                      (:entities @state))}]])
         (style/create-form-label "Define Expression")
-        (style/create-text-field {})]])}])
+        (style/create-text-field {:ref "expressionname" :defaultValue "this"})]
+
+       [:div {:style {:fontSize "106%" :lineHeight 1 :textAlign "center"}}
```

Comment:
hmm ... I happen to like the full-width button! But it seems nobody else does :)